README.md

```diff
@@ -294,6 +299,27 @@ delta a.txt b.txt
 diff -u a.txt b.txt | delta
 ```
 
+#### Environment
+
+##### a) Delta
+Delta respects the setting of some environment variables to derive the default pager to use. These are in descending order of priority :
+- `DELTA_PAGER`,
+- `BAT_PAGER` and
```

Comment:
What's your opinion about the way delta respects `BAT_PAGER`? Maybe we should start saying that it is deprecated and delta users should use `DELTA_PAGER` or `PAGER`. I may have introduced some confusion with all the connections to bat; see e.g. https://github.com/dandavison/delta/issues/326#issuecomment-718708442

Reply:
+1

I'd actually think it would be good if you'd abandon the use of the variable at some time.

Reply:
Thanks for the second opinion there.

OK, so maybe let's remove all mention of `BAT_PAGER` from the main instructions and just have a footnote / final sentence somewhere saying something like

> Delta does currently respect the `BAT_PAGER` environment variable, but that is deprecated: please use `DELTA_PAGER` or `PAGER` instead.

Reply:
cc @MarcoIeni ref #116 it doesn't necessarily affect your refactoring work but just wanted to mention to you that we're planning on making delta a bit more independent of bat by deprecating the usage of the `BAT_PAGER` env var to configure delta's pager.